test/benchmark/lmbench-read/config.json

```diff
@@ -1,5 +1,6 @@
 {
     "alert_threshold": "150%",
     "pattern": "Simple read:",
-    "field": "3"
+    "field": "3",
+    "linux_entry_presteps": "mount -t devtmpfs devtmpfs /dev"
```

Comment:
An even simpler fix is to add this line of command to `lmbench-read/run.sh`.

Another concern is that `/dev` is used frequently. Most programs or benchmarks would assume the existence of `/dev`. More benchmark programs will rely on it. So it seems to be more appropriate to prepare `/dev` for all benchmarks, instead of some benchmarks. So I think the mount command should be done upfront, for benchmarks.

Reply:
> An even simpler fix is to add this line of command to `lmbench-read/run.sh`.

`run.sh` currently serves both Asterinas and Linux, while `linux_entry_presteps` only serves Linux. For example, our device node under `/dev` is not working the same mechanism as Linux, for now. If we add this to `run.sh`, the benchmark in Asterinas won't work for now.

I agree the importance of `/dev` and move `mount -t devtmpfs devtmpfs /dev` directly to the front of `benchmark_entrypoint.sh`. But we may still need a `linux_entry_presteps` for specific benchmarks, for example, if we add benchmark to profile Ext2, we need to add `mount -t ext2 ext2 /ext2` for Linux only.